<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>656</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>656</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Talkeron v1.0 ( Copyright, 2016-2017, NiL)</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="maximumSize">
    <size>
     <width>1000</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="cmdPanel" native="true">
      <property name="minimumSize">
       <size>
        <width>400</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>400</width>
        <height>600</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="historyPanel" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>380</width>
           <height>460</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>380</width>
           <height>460</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="historyLabel">
            <property name="minimumSize">
             <size>
              <width>380</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>380</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>History</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTextBrowser" name="historyBrowser">
            <property name="minimumSize">
             <size>
              <width>360</width>
              <height>420</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>360</width>
              <height>420</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="consolePanel" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>360</width>
           <height>90</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>380</width>
           <height>90</height>
          </size>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QLabel" name="consoleLabel">
            <property name="minimumSize">
             <size>
              <width>380</width>
              <height>15</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>380</width>
              <height>15</height>
             </size>
            </property>
            <property name="text">
             <string>Console</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="CmdTextEdit" name="cmdEdit">
            <property name="minimumSize">
             <size>
              <width>360</width>
              <height>55</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>360</width>
              <height>55</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>14</pointsize>
              <kerning>false</kerning>
             </font>
            </property>
            <property name="acceptDrops">
             <bool>false</bool>
            </property>
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:14pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-size:16pt;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="acceptRichText">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="tasksPanel" native="true">
      <property name="minimumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>600</width>
        <height>600</height>
       </size>
      </property>
      <property name="focusPolicy">
       <enum>Qt::ClickFocus</enum>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_5">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="tasksLabel">
         <property name="minimumSize">
          <size>
           <width>560</width>
           <height>15</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>560</width>
           <height>15</height>
          </size>
         </property>
         <property name="text">
          <string>Tasks</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>560</width>
           <height>530</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>560</width>
           <height>530</height>
          </size>
         </property>
         <property name="focusPolicy">
          <enum>Qt::TabFocus</enum>
         </property>
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <property name="alignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
         </property>
         <widget class="QWidget" name="tasksScroll">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>558</width>
            <height>528</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="focusPolicy">
           <enum>Qt::TabFocus</enum>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout"/>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTalkeron">
    <property name="title">
     <string>Talkeron</string>
    </property>
   </widget>
   <addaction name="menuTalkeron"/>
  </widget>
  <action name="actionTakeron">
   <property name="text">
    <string>Takeron</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CmdTextEdit</class>
   <extends>QTextEdit</extends>
   <header>cmdtextedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
